ckan/views/group.py

```diff
@@ -507,6 +508,132 @@ def activity(id, group_type, is_organization, offset=0):
         _get_group_template(u'activity_template', group_type), extra_vars)
 
 
+def changes(id, group_type, is_organization):
+    '''
+    Shows the changes to an organization in one particular activity stream
+    item.
+    '''
+    activity_id = id
+    context = {
+        u'model': model, u'session': model.Session,
+        u'user': g.user, u'auth_user_obj': g.userobj
+    }
+    try:
+        activity_diff = get_action(u'activity_diff')(
+            context, {u'id': activity_id, u'object_type': u'group',
+                      u'diff_type': u'html'})
+    except NotFound as e:
+        log.info(u'Activity not found: {} - {}'.format(str(e), activity_id))
+        return base.abort(404, _(u'Activity not found'))
+    except NotAuthorized:
+        return base.abort(403, _(u'Unauthorized to view activity data'))
+
+    # 'group_dict' needs to go to the templates for page title & breadcrumbs.
+    # Use the current version of the package, in case the name/title have
+    # changed, and we need a link to it which works
+    group_id = activity_diff[u'activities'][1][u'data'][u'group'][u'id']
+    current_group_dict = get_action(u'organization_show')(
```

Comment:
This should use group_type as part  of the action call.
```suggestion
    current_group_dict = get_action(group_type + u'_show')(
```